lib/contact_list.rb

```diff
@@ -39,6 +39,11 @@ def get_contacts_from_file(json_file)
     current_contacts
   end
 
+  def delete_all_contacts
+    File.open("./lib/file_agenda.json", 'w') {|file| file.truncate(0) }
+    File.write("./lib/file_agenda.json", "[]")
+  end
+
   def sort_by(key)
     @list.sort_by! do |person|
```

Comment:
What would the `key` be in this case? `last_name` most likely? That would read muuuuch better :)

Reply:
I did a first test with last_name just before, and it work now :) 
With the key, the user can write any of them and have the result. Should I delete that? I thought it was good to have all the possibilities?

Reply:
ah alright, I understand what you mean :) will do!